gtests/test_nopython_transform.cpp

```diff
@@ -81,9 +80,46 @@ class DeltaFunctionTest : public ::testing::Test
     }
 };
 
+template <typename T>
+class InverseTest : public ::testing::Test
```

Comment:
I think we need unit tests in Python for the FFT code.  It may be done in separate PRs.  Could you please create an issue (or issues) to track it?